lib/tasks/message_queues.rake

```diff
@@ -0,0 +1,26 @@
+require_relative '../../email_alert_service/environment.rb'
+
+namespace :message_queues do
+  desc "Run worker to consume messages from rabbitmq"
+  task :consumer do
+    logger = EmailAlertService.config.logger
+    rabbitmq_options = EmailAlertService.config.rabbitmq
+
+    exchange_name = rabbitmq_options[:exchange]
+
+    rabbitmq_options[:queues].each do |queue|
+      logger.info "Bound to exchange #{exchange_name} on queue #{queue[:name]}"
+      begin
+        GovukMessageQueueConsumer::Consumer.new(
+          queue_name: queue[:name],
+          processor: queue[:processor].camelize.constantize.new(logger),
+          logger: logger
+        ).run
+      rescue SignalException => e
+        logger.info "Signal Exception: #{e}"
```

Comment:
Currently if email-alert-service catches an exception it will `exit` the program and upstart will automatically restart it for us. I think we should retain this behaviour here which as I understand it currently will only happen if all the queues receive the same exception at the same time.

Reply:
Thanks @thomasleese, have added an `exit 1` if we get a SignalException.

Reply:
I think we might want to do the same for `StandardError` as well.

Reply:
👍